src/core/Object3D.js

```diff
@@ -620,6 +621,11 @@ Object.assign( Object3D.prototype, EventDispatcher.prototype, {
 
 		object.matrix = this.matrix.toArray();
 
+		// SkinnedMesh specific
+
+		if ( this.bindMode !== undefined ) object.bindMode = this.bindMode;
+		if ( this.bindMatrix !== undefined ) object.bindMatrix = this.bindMatrix.toArray();
+
```

Comment:
What about `this.bindMatrixInverse`? For the sake of completeness we should also regard this property.